.github/workflows/release-prepare.yml

```diff
@@ -266,13 +271,20 @@ jobs:
           echo "::set-output name=version_compat::${VERSION_COMPAT}"
           echo "::set-output name=release_branch::${RELEASE_BRANCH}"
 
-      - name: Merge release branch to main
+      - name: Merge release branch to the target branch
         run: |
+          set -exu
           source "${HOLOCHAIN_RELEASE_SH}"
           cd "${HOLOCHAIN_REPO}"
 
-          git checkout --force -B main origin/main
-          git merge --ff-only "${RELEASE_BRANCH}"
+          # TODO: find out why this is required
+          git fetch --all
```

Comment:
i don't understand why this is necessary as there is a `git fetch --all` in a previous step which ought to already have updated the repository and make all branches known here. however, without this line the next condition will always be positive, i.e. the target branch is not found.

will look into this once more before merging but would appreciate any reviewer to help me out.d